src/libs/ReportActionComposeFocusManager.js

```diff
@@ -30,8 +32,20 @@ function clear() {
     focusCallback = null;
 }
 
+/**
+ * Tells about the current focus state of the report action composer.
```

Comment:
`Tells about the current focus...` > `Exposes the current focus...`

Reply:
Sorry, do you want to update the comment?

Reply:
Yep, sorry :)